endoreg_db/models/media/video/video_file.py

```
@@ -223,6 +223,31 @@ def active_file_url(self) -> str:
         url = _file.url
 
         return url
+    
+    @property
+    def active_raw_file(self) -> File:
+        if self.has_raw:
+            return self.raw_file
+        else:
+            raise ValueError("Has no raw file")
```

Pylint (4.0.0)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)